src/main/resources/wa-task-configuration-employment-et_englandwales.dmn

```diff
@@ -646,7 +647,7 @@ else ""</text>
           <text>"majorPriority"</text>
         </outputEntry>
         <outputEntry id="LiteralExpression_0xy9k8p">
-          <text>if (caseData != null and caseData.isUrgent != null and caseData.isUrgent = "Yes")
+          <text>if caseData != null and caseData.referralCollection != null and count(caseData.referralCollection.count) &gt; 0 and (reverse(caseData.referralCollection)[1]).value.isUrgent = "Yes"
```

Comment:
What language is this even? :/
count(caseData.referralCollection.count) looks strange to me but I'll trust you

Reply:
This is FEEL/Scala.
But you are actually right, I shouldn't have "count" twice there, I will amend.
Great spot!